platform-includes/debug-symbols-apple/_default.mdx

```diff
@@ -122,8 +123,12 @@ else
 echo "error: sentry-cli not installed, download from https://github.com/getsentry/sentry-cli/releases"
 fi
 ```
+<Note>
+You can add `--force-foreground` argument to sentry-cli to help you debug any error that could happen during the upload process.
+</Note>
+
 
-3.  You also need to add the following line to the _Input Files_ section in the _Run Script_ from step 2:
+4.  You also need to add the following line to the _Input Files_ section in the _Run Script_ from step 2:
```

Comment:
Can we move this step up to before step 3 (switch steps 3 & 4)? I think this step feels pretty disjointed from the rest and can be easily missed.

Reply:
Yeah, makes sense.